code/modules/mining/minebot.dm

```diff
@@ -280,10 +280,11 @@
 
 /obj/item/mine_bot_upgrade/proc/upgrade_bot(mob/living/simple_animal/hostile/mining_drone/M, mob/user)
 	if(M.melee_damage_upper != initial(M.melee_damage_upper))
-		to_chat(user, "<span class='warning'>[src] already has a combat upgrade installed!</span>")
+		to_chat(user, "<span class='warning'>[M.name] already has a combat upgrade installed!</span>")
```

Comment:
You don't need to specify `.name` and this can just be `[M]`

While you're here change the name to something more sensible that isn't a single character.

Reply:
Oh, I wasn't sure if I could do that or not. Neat!